health-services/project-factory/src/server/service/dataManageService.ts

```
@@ -30,13 +31,38 @@
 
     const type = request.query.type;
     // Get response data from the database
-    const responseData = await searchGeneratedResources(request);
+    var responseData = await searchGeneratedResources(request);
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)